app/assets/stylesheets/_commons.css.scss

```diff
@@ -222,3 +222,25 @@ body {
   .tab-bar .menu-icon { text-indent: 0.6rem; }
   #squiggle { fill: $bright-green; }
 }
+
+.social-icons {
+  a + a { margin-left: 1rem; }
+
+  i {
+    background: $bright-green;
+    border-radius: 35px;
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.